ArduPlane/Plane.h

```diff
@@ -563,9 +563,10 @@ class Plane : public AP_Vehicle {
         // altitude adjustments
         float target_alt = -1;   // don't default to zero here, as zero is a valid alt.
         uint32_t last_target_alt = 0;
+        Location::AltFrame last_target_alt_frame = Location::AltFrame::ABSOLUTE;
```

Comment:
This is unused.

Reply:
In this PR yes, but when it's used in my other PR this was extracted from.

Reply:
> Should I move this back over there?

Yes please.

Reply:
Turns out I do need to use it to fix the CI fail